hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -304,6 +304,24 @@ public HoodieRecord generateUpdateRecord(HoodieKey key, String commitTime) throw
     return updates;
   }
 
+  public List<HoodieRecord> generateUpdatesWithDiffPartition(String commitTime, List<HoodieRecord> baseRecords)
```

Comment:
lets keep this with the actual test that uses this? only thing that is needed here seems like the `partitionPaths`?

Reply:
this method uses some variables in HoodieTestDataGenerator. if we move this to the test class, then have to pass in DataGenerator class and hence left it here. If you still persist, I can move it. 

Yes, you are right. To choose the new partition chose, I alter between two partitions.